lib/PEX.ts

```diff
@@ -48,30 +51,27 @@ export class PEX {
    */
   public evaluatePresentation(
     presentationDefinition: IPresentationDefinition,
-    presentation: IPresentation | JwtDecodedVerifiablePresentation | string,
+    presentation: IPresentation | CompactJWT | string,
```

Comment:
Huh, you removed a decoded VP with CompactJWT which is a string.
So now this reads as `IPresentation | string | string` basically

Reply:
fixed